themes/phenomic-theme-base/src/components/Container/index.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component, PropTypes } from "react"
+
+import styles from "./index.css"
+
+export default class Container extends Component {
+
+  static propTypes = {
+    children: PropTypes.oneOfType([ PropTypes.array, PropTypes.object ]),
+  };
+
+  render() {
+    return (
+      <div className={ styles.container }>
```

Comment:
I think this is unnecessary, can we just move the styling part to AppContainer ?